docs/specs/6-auth-via-aad.md

```diff
@@ -0,0 +1,95 @@
+
+---
+layout: docs
+toc: true
+title: Server SDK Design Spec
+group: specs
+---
+
+# Auth via Azure Active Directory

```

Comment:
So there are 2 auth for AAD, one is for REST API one is for WebSocket client. Shall we cover both?